src/pcapi/local_providers/titelive_things/titelive_things.py

```diff
@@ -316,3 +324,12 @@ def get_extra_data_from_infos(infos: dict) -> dict:
     if infos["commentaire"] != "":
         extra_data["comment"] = trim_with_elipsis(infos["commentaire"], 92)
     return extra_data
+
+
+def is_unreleased_book(product_info: dict) -> None:
+    return (
+        "titre" in product_info
+        and product_info["titre"].lower() == UNRELEASED_BOOK_MARKER
+        and "auteurs" in product_info
+        and product_info["auteurs"].lower() == UNRELEASED_BOOK_MARKER
+    )
```

Comment:
```suggestion
    return product_info.get("titre", "").lower() == product_info["auteurs"].lower() == UNRELEASED_BOOK_MARKER
```

ou bien en 3 lignes (un peu plus lisible, peut-être) :

```suggestion
    title = product_info.get("titre", "").lower()
    author = product_info.get("auteurs", "").lower()
    return title == author == UNRELEASED_BOOK_MARKER
```

Reply:
Je plussoie la seconde suggestion.